### ISSUE TITLE
Enum EmptyLineSeparator

### ISSUE THREAD

**Issue by Lranger321** [checkstyle/checkstyle#10529]
The error will not be displayed, maybe i have problem with configuration file ? 
```
<module name="EmptyLineSeparator">
            <property name="allowMultipleEmptyLines" value="false"/>
            <property name="allowMultipleEmptyLinesInsideClassMembers" value="false"/>
            <property name="allowNoEmptyLineBetweenFields" value="true"/>
</module>
```
```
public enum Bar {
  FOO(""),
  BAR("");
  
  



  private String a;
}
```
I tried with Maven Checkstyle plugin 3.1.2 which uses 8.43 and also tried overriding its version to the latest Checkstyle

**Comment by strkkk** [comment]
will be fixed in https://github.com/checkstyle/checkstyle/issues/11241, @Lranger321 sorry that we missed this before.

**Comment by pavelicii** [comment]
This can be closed now via #11245.

**Comment by strkkk** [comment]
fixed in https://github.com/checkstyle/checkstyle/issues/11241
